src/pages/Home/Home.js

```diff
@@ -0,0 +1,67 @@
+import './Home.css';
+import axios from "axios";
+import {useEffect, useState} from "react";
+import {Link} from "react-router-dom";
+import NumberFormat from "../../helpers/NumberFormat";
+import LongTitlesTruncate from "../../helpers/LongTitelsTruncate";
+
+function Home() {
+    const [reddits, setReddits] = useState(null);
```

Comment:
Ik zou een lege array als default value pakken, dan ben je verderop op regel 42 de undefined check kwijt
```suggestion
    const [reddits, setReddits] = useState([]);
```